docker/testrunner/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y sudo openjdk-7-jdk xorg xvfb xserver-xorg-video-dummy
+RUN useradd -m -G users testrunner
+ADD entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
Is this necessary? I thought docker would keep the permissions of a file when using the `ADD` instruction.